app/models/flatirons/saas/subscription.rb

```diff
@@ -48,5 +49,12 @@ def create_stripe_subscription
     def update_stripe_subscription
       stripe_service.update_subscription(stripe_subscription_id, stripe_price_id) if stripe_price_id_changed?
     end
+
+    def destroy_stripe_subscription
+      stripe_service.delete_subscription(stripe_subscription_id)
```

Comment:
We will need a way to define what should be the cancelation mode on stripe.

Like: prorate or invoice_now.
Maybe we can put it on the subscritable options like:

```ruby
class User
  subscriptable proate_on_cancel: true #or
  subscriptable invoice_now_on_cancel: true 
end
```
So you can access this options using the subscritable model like:

```ruby
 def destroy_stripe_subscription
      invoice_now = subscriptable.subscriptable_options[:invoice_now_on_cancel]
      prorate = subscriptable.subscriptable_options[:prorate_on_cancel]
      stripe_service.delete_subscription(stripe_subscription_id, invoice_now: invoice_now, prorate: prorate)
 end
```